var/spack/repos/spack_repo/builtin/packages/paraview/package.py

```diff
@@ -238,6 +238,10 @@ class Paraview(CMakePackage, CudaPackage, ROCmPackage):
 
     depends_on("qt@:4", when="@:5.2.0+qt")
     depends_on("qt+sql", when="+qt")
+
+    # https://discourse.paraview.org/t/paraview-5-9-and-minimum-recommended-qt-version/5333
+    depends_on("qt@5.12:5 +sql", when="+qt @5.9:")
```

Comment:
The ordering of this constraint is a bit vague at first glance, can we have the paraview version constraint first? Similar to the constraints above.

@kwryankrattiger I assume we need to bound this for Paraview 6?

Reply:
As we have not heard from @kwryankrattiger I have bounded this at paraview < 6, via the last 5.x of 5.13